VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/hearing-controls/hearing-controls-base.component.ts

```diff
@@ -245,6 +251,17 @@ export abstract class HearingControlsBaseComponent implements OnInit, OnDestroy
         this.displayConfirmPopup = true;
     }
 
+    displayConfirmLeaveDialog() {
```

Comment:
this too i think can be replaced with the other leave modal code built into the `consultation service` `displayConsultationLeaveModal()`